crates/kitsune_p2p/kitsune_p2p/Cargo.toml

```diff
@@ -44,7 +44,7 @@ thiserror = "1.0.22"
 tokio = { version = "1.27", features = ["full"] }
 tracing = "0.1"
 tokio-stream = "0.1"
-tx5 = { version = "=0.0.1-alpha.20", optional = true }
+tx5 = { version = "=0.0.2-alpha", optional = true }
```

Comment:
Please update all the tx5 references
- [hc_run_local_services](https://github.com/holochain/holochain/pull/2660/files#diff-cbb293afce321814fdfa81cc8eb95e0c49cf0c1f4b9c6e460219e55e5b45c13b)
- [kitsune_p2p](https://github.com/holochain/holochain/pull/2660/files#diff-accbdb1d798d57fe2faf13b8a6e2120172c4fa19c4990bcfc1f70e7ea0a72e75)
- [holochain](https://github.com/holochain/holochain/pull/2660/files#diff-8c72bca6e5c9cacad5ed1706bcd7a0d9d2a2344fc548c21c826fa20cd4a4309f)

Reply:
Yes, done